src/Runner/TestHandler.php

```diff
@@ -113,7 +113,7 @@ private function initiatePhpVersion($version, PhpInterpreter $interpreter)
 
 	private function initiatePhpIni($value, PhpInterpreter $interpreter)
 	{
-		$interpreter->arguments .= ' -d ' . Helpers::escapeArg($value);
+		$interpreter->addArguments(' -d ' . Helpers::escapeArg($value));
```

Comment:
Have you considered moving the logic for arguments handling to interpreters, i.e. sth. like

~~~php
$interpreter->setPhpIniPath($value);
~~~